### PR TITLE
x11-terms/cool-retro-term: Bump to 1.2.0 and 9999 version Ebuild

### PR DESCRIPTION
**Pull request by ThamognyaKodi**
Changes:
- added 1.2.0 version ebuild
- added 9999 version ebuild
- Added Manifest updates
- Requesting to be a proxy-maint or proxy-maintainer or maintainer for this package `x11-terms/cool-retro-term`

Signed-off-by: Thamognya Kodi <contact@thamognya.com>

### CODE REVIEW

File: x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Gentoo Authors
```
Comment (ConiKost) [inline x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild:1]: What about existing copyright, which is present in old ebuild?

File: x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+DESCRIPTION="terminal emulator which mimics the look and feel of the old cathode tube screens"
+HOMEPAGE="https://github.com/Swordfish90/cool-retro-term"
+
+SRC_URI="https://github.com/Swordfish90/cool-retro-term/archive/${PV}.tar.gz -> ${P}.tar.gz
+		https://github.com/Swordfish90/qmltermwidget/archive/${QTW_PV}.tar.gz -> ${QTW_P}.tar.gz"
+
+LICENSE="GPL-2 GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE=""
```
Comment (ConiKost) [inline x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild:21]: If empty, drop it.

File: x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	dev-qt/qtquickcontrols:5[widgets]
+	dev-qt/qtsql:5
+	dev-qt/qtwidgets:5
+"
+
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+	rmdir ${WORKDIR}/${P}/${QTW_PN}
```
Comment (ConiKost) [inline x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild:36]: Missing ```|| die```.

File: x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	mv "${WORKDIR}/${QTW_P}" ${WORKDIR}/${P}/${QTW_PN}
```
Comment (ConiKost) [inline x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild:37]: Missing ```|| die```.

File: x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+	pushd qmltermwidget || die
+	eapply "${FILESDIR}"/qmltermwidget-0.2.0-gcc-10.patch
```
Comment (ConiKost) [inline x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild:39]: Can't be this done through regular ```PATCHES=()``` variable?

File: x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+	popd || die
+}
+
+src_configure() {
+	eqmake5 PREFIX="/usr/bin"
```
Comment (ConiKost) [inline x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild:44]: Prefix shouldn't be hardcoded, use ```${EPREFIX}```.

File: x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+src_install() {
+	emake
+	mv "${WORKDIR}/${P}/${PN}" /usr/bin/${PN}
```
Comment (ConiKost) [inline x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild:49]: Missing ```|| die```.

File: x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+pkg_postinst() {
+	xdg_icon_cache_update
+	xdg_desktop_database_update
+}
+
+pkg_postrm() {
+	rm -rf "/usr/bin/${PN}"
```
Comment (ConiKost) [inline x11-terms/cool-retro-term/cool-retro-term-1.2.0-r2.ebuild:58]: He? Why? (And also Missing ```|| die```.)

**Review by ConiKost — CHANGES_REQUESTED** [review]

**Comment by ThamognyaKodi** [comment]
Thank you for the feedback. I will try to fix it and the. Make another PR.

**Comment by juippis** [comment]
In future you can just squash your commits and force-push. I see you're already working in a git branch so it should be rather easy.

**Pull request closed without merging**